cli/endpoints/online/managed/vnet/setup/main.bicep

```diff
@@ -0,0 +1,137 @@
+// Execute this main file to configure Azure Machine Learning end-to-end in a moderately secure set up
+
+// Parameters
+@minLength(2)
+@maxLength(10)
+@description('Prefix for all resource names.')
+param suffix string
+
+@description('Azure region used for the deployment of all resources.')
+param location string = resourceGroup().location
+
+@description('[Optional] Azure region used for appinsights.')
+param appinsightsLocation string = location
+
+@description('Set of tags to apply to all resources.')
+param tags object = {}
+
+@description('Virtual network address prefix')
+param vnetAddressPrefix string = '192.168.0.0/16'
```

Comment:
Looks like we are creating only subnet for both scoring and for private endpoints. Expected?

Reply:
Wanted to keep it simple for this sample. Let me know if this needs to be changed